src/components/cardbuilder/cardBuilder.js

```diff
@@ -530,6 +538,8 @@ define(['datetime', 'imageLoader', 'connectionManager', 'itemHelper', 'focusMana
 
                 imgUrl = apiClient.getScaledImageUrl(item.Id, {
                     type: "Primary",
+                    maxHeight: height,
+                    maxWidth: width,
```

Comment:
Why do some of them get the height as well? Shouldn't one of them be enough?

Reply:
I think it's for everything that doesn't fall into anything else. Doesn't fall into any specific category, so just display whatever, I think ?